tests/jsmarionette/runner/marionette-js-runner/bin/marionette-mocha

```diff
@@ -151,9 +151,53 @@ function main(argv) {
 
   var reporter = new args[0].reporter(runner.mochaRunnerProxy);
 
-  runner.process.stdout.pipe(process.stdout);
-  runner.process.stderr.pipe(process.stderr);
-  runner.process.on('exit', process.exit);
+  // make sure we only output complete lines to stdout and stderr to ensure
+  // cohesive logs.
+  function logLine(outputData, isError) {
+    // direct to the appropriate output.
+    var _log = !!isError ? console.error : console.log;
```

Comment:
nit: No need for `!!` since `?` operator coerces the thing into a boolean anyway